packages/otel/tsdown.config.ts

```diff
@@ -1,17 +1,12 @@
-import { argv } from 'node:process'
-
-import { defineConfig } from 'tsup'
+import { defineConfig } from 'tsdown'
 
 export default defineConfig([
   {
     clean: true,
     format: ['cjs', 'esm'],
     entry: ['src/bootstrap/main.ts', 'src/main.ts'],
     tsconfig: 'tsconfig.json',
-    splitting: false,
```

Comment:
maybe related to https://github.com/rolldown/tsdown/issues/317 ? @TheAlexLichter

Reply:
we turned off splitting because of [a packaging issue in deno 2](https://github.com/netlify/primitives/pull/85) that we should probably investigate if we're changing that

Reply:
Okay, let's ensure that this packaging issue is resolved by now (or we can adapt the config to avoid code splitting with a workaround)